### PR TITLE
bugfix for issues#83

### PR DESCRIPTION
**Pull request by kobusan**
Hi !  

this PR is bugfix for https://github.com/heavysixer/node-pptx/issues/83

https://github.com/heavysixer/node-pptx/issues/83#issuecomment-1094073275

Please consider it.
Thx. :)

### CODE REVIEW

**Comment by gustaflunnergard** [comment]
Would be great to fix this!

**Review by jankapunkt — APPROVED** [review]

**Comment by jankapunkt** [comment]
@heavysixer can you please merge this and create a new release? Can you otherwise please add one co-maintainer?

**Review by jboler — APPROVED** [review]
Merging for now to unblock this.

It's a mystery to me how `zip.files.hasOwnProperty(key)` could return `TypeError: a.hasOwnProperty is not a function` because `zip.files` is [initialized to `{}`](https://github.com/Stuk/jszip/blob/v3.6.0/lib/index.js#L22) and [`zip.forEach()` also calls `zip.files.hasOwnProperty()`](https://github.com/Stuk/jszip/blob/v3.6.0/lib/object.js#L182-L185).

Could someone who can reproduce the error do a `console.log(zip.files)` and determine why the error is occurring?

**Pull request merged**
